packages/opencensus-nodejs/package.json

```diff
@@ -55,8 +55,7 @@
   },
   "dependencies": {
     "@opencensus/core": "^0.0.11",
-    "@opencensus/instrumentation-all": "^0.0.11",
-    "extend": "^3.0.2",
-    "require-in-the-middle": "^4.0.0"
+    "@opencensus/nodejs-base": "^0.0.11",
+    "@opencensus/instrumentation-all": "^0.0.11"
```

Comment:
Nit: Move `@opencensus/instrumentation-all` above `@opencensus/nodejs-base` to keep the ordering intact.